WRITING.md

```diff
@@ -154,6 +155,67 @@ This will render **Astro's latest version: v1.2.1**.
 
 The `<Version />` component is currently used in our Integrations pages as a way to keep each integration’s version up-to-date without having to merge changes to these pages directly. It's worth noting that this component is only updated at build-time, thus a package's version will not change until the site is rebuilt, be it manually or because a new PR was merged into main.
 
+### Tabs Components
+
+For pieces of content relevant to only a percentage of users (e.g. specific package manager or UI framework guidance), we have Tabs components to help you.
+
+For the two common cases: package managers and UI frameworks, the corresponding `<PackageManagerTabs>` and `<UIFrameworkTabs>` components already got you covered.
+
+It's worth mentioning that these components share state, so if you change the active tab of one `<PackageManagerTabs>` component, all of the other package manager Tabs components on the same page will also change.
+
+#### Examples
+
+All you have to do to use a Tabs component is to import it into the file and fill the available slots with content, for example:
```

Comment:
``````suggestion
To use an existing Tabs component (e.g. `<PackageManagerTabs>` , `<UIFrameworkTabs>`), import it into the `.md` page's frontmatter using the `setup:` property. 

```markdown
---
setup: | 
  import InstallGuideTabGroup from '~/components/TabGroup/InstallGuideTabGroup.astro';
  import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
---
```

Then, create a `<Fragment>` for each tab. The fragment's slot name will identify the tab label and the content between the opening and closing `<Fragment>...</Fragment>` tags will be the panel content.

Here is an example of `<PackageManagerTabs>` showing the `create astro` commands from our [Automatic Astro Installation](https://docs.astro.build/en/install/auto/#1-run-the-setup-wizard) page.
``````